src/content/app/species/SpeciesPage.module.css

```diff
@@ -11,11 +11,21 @@
   justify-items: start;
   line-height: 1;
   max-width: 1380px;
-  padding-left: 30px;
+}
+
+.topbar_withoutSidebarNavigation {
+  display: grid;
+  align-items: center;
```

Comment:
I don't think you are using any grid features for this element; so the two CSS rules above are probably unnecessary.